lib/templates/webmanifest.js

```diff
@@ -6,8 +6,9 @@ const renderWebManifest = (name, color) => `
 	"background_color": "${color}",
 	"display": "standalone",
 	"orientation": "portrait",
-	"start_url": "https://${name}.gradebook.app/my/dashboard",
+	"start_url": "https://${name}.gradebook.app/my/dashboard?utm_source=homescreen",
```

Comment:
I've seen `ref` as a more modern alternative to `utm_source`. Thoughts? Also think we could use `app` as the ref?

Reply:
I don't care either way, `ref` is a little cleaner and it's just for us to track anyway.